node_modules/oae-core/topnavigation/topnavigation.html

```diff
@@ -127,6 +127,9 @@ <h1 id="topnavigation-target" class="sr-only">__MSG__SKIP_MENU_TARGET__</h1>
                                         <button id="topnavigation-signin-button" class="btn btn-primary pull-right" type="submit">__MSG__SIGN_IN__</button>
                                         <span id="topnavigation-signin-signing-in" class="hide" class="pull-right">__MSG__SIGNING_IN__...</span>
                                         {if oae.api.config.getValue("oae-authentication", "local", "allowAccountCreation")}
+											<span>__MSG__FORGET_PASSWORD__
```

Comment:
I think this will need to be moved to `signin.html` due to other changes made around the login UI.